release.sh

```diff
@@ -65,6 +65,12 @@ while [[ $# -gt 0 ]]; do
     rpm-sync "${HOTFIXDIR}/rpm/index.yaml" "${BUILDDIR}/rpm"
   fi
 
+  if [[ -f "${HOTFIXDIR}/rpm/.createrepo" ]]; then
```

Comment:
I could undo this, I left it in for now incase it was useful for something else later.